Author: Milan P. StaniÄ‡ <mps@arvanta.net>
Date: 2024-01-08 16:37:48.000000000 +0100

Alpine doesn't have tinfo lib so build tools/kwboot with ncursesw

--- a/tools/Makefile	2024-01-08 16:37:48.000000000 +0100
+++ b/tools/Makefile	2024-04-12 11:00:20.921739870 +0200
@@ -200,7 +200,7 @@
 HOSTCFLAGS_kwboot.o += -pthread
 HOSTLDLIBS_kwboot += -pthread
 HOSTLDLIBS_kwboot += \
-	$(shell pkg-config --libs tinfo 2> /dev/null || echo "-ltinfo")
+	$(shell pkg-config --libs ncursesw 2> /dev/null || echo "-lncursesw")
 
 ifdtool-objs := $(LIBFDT_OBJS) ifdtool.o
 hostprogs-$(CONFIG_X86) += ifdtool
